nsv13/code/modules/clothing/custom_clothes.dm

```diff
@@ -249,6 +249,15 @@
 	item_state = "sun"
 	dynamic_hair_suffix = ""
 
+/obj/item/clothing/accessory/bomber_jacket_accessory
+	name = "Thin bomber jacket"
```

Comment:
```suggestion
	name = "thin bomber jacket"
```

Reply:
the underscores make sense, but why make the name lowercase only?